cmake/boost.cmake

```diff
@@ -1,13 +1,35 @@
+# Copyright (C) 2008-2019 LAAS-CNRS, JRL AIST-CNRS, INRIA
 #
-# @file boost.cmake
-# @author Maximilien Naveau (maximilien.naveau@gmail.com)
-# @copyright Copyright (c) 2019, New York University and Max Planck Gesellschaft.
-# @license License BSD-3 clause
-# @date 2019-05-06
-# 
-# @brief This file is copied/inspired from
-# https://github.com/jrl-umi3218/jrl-cmakemodules/blob/master/boost.cmake
-# 
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
```

Comment:
If this is GPL, it probably means that all cmake-files including it need to be GPL as well...

Reply:
so what do we do? They say it's free of copying... And I let the licence inside the file.
what can I do more?